src/lib/messenger.js

```diff
@@ -34,7 +34,8 @@ export default function messenger(bot) {
       } else if (inputParser.isAskingForBoegivar(text)) {
         handleBoegivar(bot, message.chat, user.firstName);
       } else if (inputParser.isAskingForTourettes(text)) {
-        handleTourettes(bot, message.chat, text);
+        const commandText = text.substr(11, text.length);
```

Comment:
In general it's desired to have a function only do one thing, so in this case having a function that strips the command should take a text for input, and return a new text. The new text should then be the input text without the command if found. 

It makes for much easier-to-reason-about code, see: [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle)